backend/internal/oauth/oauth2/token/tokenhandler.go

```diff
@@ -197,7 +218,7 @@ func (th *TokenHandler) HandleTokenRequest(w http.ResponseWriter, r *http.Reques
 	}
 
 	// Log successful token generation.
-	logger.Info("Token generated successfully", log.String("client_id", clientID))
+	logger.Info("Token generated successfully", log.String("client_id", clientID), log.String("grant_type", grantType))
```

Comment:
shall we make this debug?

Reply:
Fixed with a6832baeaae66e0b2d0eb0a4feef841aa13c2acd